src/index.mjs

```diff
@@ -283,25 +287,24 @@ async function loop() {
     var elapsedTime = performance.now() - clockStart;
 
     // Draw face overlay
-    if( webgazer.params.showFaceOverlay )
-    {
+    if (webgazer.params.showFaceOverlay) {
       // Get tracker object
       var tracker = webgazer.getTracker();
-      faceOverlay.getContext('2d').clearRect( 0, 0, videoElement.videoWidth, videoElement.videoHeight);
+      faceOverlay.getContext('2d').clearRect(0, 0, videoElement.videoWidth, videoElement.videoHeight);
       tracker.drawFaceOverlay(faceOverlay.getContext('2d'), tracker.getPositions());
     }
 
     // Feedback box
     // Check that the eyes are inside of the validation box
-    if( webgazer.params.showFaceFeedbackBox )
-      checkEyesInValidationBox();
+    if (webgazer.params.showFaceFeedbackBox)
+      params.faceInFrame = checkEyesInValidationBox();
```

Comment:
I'm not sure if the parameters is the correct place for this -- it seems more like a flag than a preset parameter. Maybe we can just let the user call checkEyesInValidationBox() freely if they can't already?